collections/nemo_nlp/nemo_nlp/data/text_data_layers.py

```diff
@@ -7,6 +17,16 @@
 
 
 class TextDataLayer(DataLayerNM):
+    """
```

Comment:
This docstring format looks wrong. Please refer to https://github.com/NVIDIA/NeMo/blob/master/collections/nemo_asr/nemo_asr/data_layer.py#L18 for example